partials/information-banner.php

```diff
@@ -39,3 +39,38 @@
         <?php
     }
 }
+
+// Page-specific banner
+$enable_banner_on_single_view = hale_get_acf_field_status('enable_banner_on_single_view');
+$show_banner = get_field('show_post_banner');
+
+if ($enable_banner_on_single_view && $show_banner) {
+    $banner_content = "";
+    // Main text
+    $banner_text = get_field('post_banner_text');
+    if ($banner_text && $banner_text != "") {
+        $banner_content .= "<p class='govuk-body page-banner__text'>$banner_text</p>";
+    }
+
+    // Get max number of links
+    $number_of_banner_links = hale_get_post_type_setting('single_view_banner_max_links');
+
+    if (!$number_of_banner_links) $number_of_banner_links = 0;
```

Comment:
If you want to make clear it is evaluating a boolean, you could do it like `$number_of_banner_links = ( isset( $number_of_banner_links ) && $number_of_banner_links ) ? $number_of_banner_links : 0;` Although I appreciate it isn't as concise.

Reply:
Done